irods_python_ci_utilities/irods_python_ci_utilities.py

```diff
@@ -55,30 +55,31 @@ def subprocess_get_output(*args, **kwargs):
 
 def install_os_packages_apt(packages):
     args = ['sudo', 'apt-get', 'install', '-y'] + list(packages)
-    subprocess_get_output(args, check_rc=True)
+    return subprocess_get_output(args, check_rc=True)
 
 def install_os_packages_yum(packages):
     args = ['sudo', 'yum', 'install', '-y'] + list(packages)
-    subprocess_get_output(args, check_rc=True)
+    return subprocess_get_output(args, check_rc=True)
 
 def install_os_packages_zypper(packages):
     args = ['sudo', 'zypper', '--non-interactive', 'install'] + list(packages)
-    subprocess_get_output(args, check_rc=True)
+    return subprocess_get_output(args, check_rc=True)
 
 def install_os_packages(packages):
     dispatch_map = {
         'Ubuntu': install_os_packages_apt,
         'Centos': install_os_packages_yum,
         'Centos linux': install_os_packages_yum,
         'Opensuse ': install_os_packages_zypper,
+        'Opensuse leap': install_os_packages_zypper,
     }
     try:
-        dispatch_map[get_distribution()](packages)
+        return dispatch_map[get_distribution()](packages)
     except KeyError:
         raise_not_implemented_for_distribution()
 
 def install_os_packages_from_files_apt(files):
-    '''files are installed individually in the order supplied, so inter-file dependencies must be handled by the caller'''
+    #'''files are installed individually in the order supplied, so inter-file dependencies must be handled by the caller'''
```

Comment:
what does this # do?

Reply:
I was under the impression that string was supposed to be a comment.

Reply:
sure, so no need for the triple quotes, right?